source/conf.py

```diff
@@ -58,6 +59,14 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = []
 
+# Cross-refering other projects
+# https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html
+intersphinx_mapping = {
+    "seis": ("https://seismo-learn.org/seismology/", None),
+    "seis101": ("https://seismo-learn.org/seismology101/", None),
+    "software": ("https://seismo-learn.org/software/", None),
```

Comment:
This project doesn't need the `software` intersphinx_mapping setting, but I'd like to keep it so that we can have consistent/similar `conf.py` in all the three projects.

Reply:
sound good